.github/workflows/stepci-2.8.2.yml

```
@@ -0,0 +1,27 @@
+name: stepci/stepci@2.8.2
+on:
+  pull_request:
+    paths:
+      - .github/workflows/reusable-docker-workflow.yml
+      - .github/workflows/stepci-2.8.2.yml
+  push:
+    branches:
+      - '*'
+    paths:
+      - .github/workflows/reusable-docker-workflow.yml
+      - .github/workflows/stepci-2.8.2.yml
+  schedule:
+    - cron: "16 1 * * *"
```

I know this is not ideal (see #2)... this one should be:


```suggestion
    - cron: "25 1 * * *"
```